tools/ColonCatcher/2017-08-09-colon_operator_log.txt

```diff
@@ -0,0 +1,53 @@
+..code\controllers\subsystem\shuttle.dm Lines: 402 Total Colons: 1
+..code\datums\ai_laws.dm Lines: 237 Total Colons: 1
+..code\datums\browser.dm Lines: 134, 135, 138, 141 Total Colons: 6
+..code\datums\datumvars.dm Lines: 342 Total Colons: 1
+..code\datums\mind.dm Lines: 805, 806, 816, 821 Total Colons: 5
+..code\datums\riding.dm Lines: 45 Total Colons: 1
+..code\datums\antagonists\datum_clockcult.dm Lines: 68, 69 Total Colons: 2
+..code\game\communications.dm Lines: 295 Total Colons: 3
+..code\game\gamemodes\objective.dm Lines: 847 Total Colons: 1
+..code\game\machinery\newscaster.dm Lines: 364, 415, 428, 478, 944 Total Colons: 5
+..code\game\machinery\overview.dm Lines: 99, 245 Total Colons: 2
+..code\game\machinery\robot_fabricator.dm Lines: 21, 23, 24, 25, 28 Total Colons: 5
+..code\game\machinery\status_display.dm Lines: 153 Total Colons: 4
+..code\game\machinery\computer\communications.dm Lines: 504, 635 Total Colons: 2
+..code\game\machinery\computer\medical.dm Lines: 62, 67 Total Colons: 2
+..code\game\machinery\computer\security.dm Lines: 121, 128, 209, 229, 385, 401 Total Colons: 6
+..code\game\machinery\doors\brigdoors.dm Lines: 193 Total Colons: 4
+..code\game\mecha\mecha.dm Lines: 717 Total Colons: 1
+..code\game\mecha\equipment\tools\medical_tools.dm Lines: 110, 293, 435, 454 Total Colons: 4
+..code\game\mecha\equipment\tools\work_tools.dm Lines: 371 Total Colons: 1
+..code\game\objects\items\devices\scanners.dm Lines: 145 Total Colons: 1
+..code\game\objects\items\devices\transfer_valve.dm Lines: 103 Total Colons: 1
+..code\game\objects\items\devices\PDA\cart.dm Lines: 263, 393, 410, 701, 702, 703, 706, 707, 708 Total Colons: 9
+..code\modules\admin\admin.dm Lines: 41, 137, 144, 156, 261, 307, 321, 367 Total Colons: 9
+..code\modules\admin\sql_message_system.dm Lines: 268 Total Colons: 2
+..code\modules\admin\stickyban.dm Lines: 188 Total Colons: 1
+..code\modules\admin\DB_ban\functions.dm Lines: 87, 88, 89, 315, 316, 317 Total Colons: 6
+..code\modules\admin\verbs\adminjump.dm Lines: 91 Total Colons: 1
+..code\modules\admin\verbs\debug.dm Lines: 321, 338, 464, 467 Total Colons: 4
+..code\modules\admin\verbs\diagnostics.dm Lines: 82 Total Colons: 4
+..code\modules\admin\verbs\randomverbs.dm Lines: 847, 852, 858, 864, 870, 876, 882, 888, 894, 900, 906, 912, 918, 924, 930, 936, 942, 948 Total Colons: 18
+..code\modules\admin\verbs\SDQL2\SDQL_2.dm Lines: 392, 434 Total Colons: 2

```

Comment:
both of these are in strings
https://github.com/tgstation/tgstation/blob/master/code/modules/admin/verbs/SDQL2/SDQL_2.dm#L392
https://github.com/tgstation/tgstation/blob/master/code/modules/admin/verbs/SDQL2/SDQL_2.dm#L434

Not bothered to check other files, this is enough to prove it's still a bit wonk.